.github/workflows/build.yml

```diff
@@ -5,11 +5,21 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - name: Failure Slack Report
+      if: always()
+      uses: ravsamhq/notify-slack-action@master
+      with:
+        status: ${{ job.status }}
+        notify_when: 'failure'
```

Comment:
Will this notify when running in branches?

Reply:
Updated the if block so it should only run on `master` (https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#determining-when-to-use-contexts).